keel-core/src/test/kotlin/com/netflix/spinnaker/keel/services/ResourceHistoryServiceTests.kt

```diff
@@ -70,124 +84,176 @@ class ResourceHistoryServiceTests : JUnit5Minutests {
       }
 
       test("can get resource summary by delivery config") {
-        val summaries = resourceHistoryService.getResourceSummariesFor(deliveryConfig)
+        val summaries = subject.getResourceSummariesFor(deliveryConfig)
 
         expect {
           that(summaries.size).isEqualTo(1)
           that(summaries.map { it.status }.filter { it == CREATED }.size).isEqualTo(1)
         }
       }
-    }
 
-    context("enriched resource event history") {
-      context("with application paused at various times after resource creation") {
-        before {
-          repository.storeResource(resource)
-          repository.appendResourceHistory(ResourceCreated(resource, clock))
-          clock.incrementBy(TEN_MINUTES)
-          repository.appendResourceHistory(ResourceValid(resource, clock))
-          clock.incrementBy(TEN_MINUTES)
-          actuationPauser.pauseApplication(resource.application, user)
-          clock.incrementBy(TEN_MINUTES)
-          actuationPauser.resumeApplication(resource.application, user)
-          clock.incrementBy(TEN_MINUTES)
-          actuationPauser.pauseApplication(resource.application, user)
-        }
-
-        test("has application paused and resumed events injected in the right positions") {
-          val events = resourceHistoryService.getEnrichedEventHistory(resource.id)
-          expectThat(events)
-            .map { it.javaClass }
-            .containsExactly(
-              ApplicationActuationPaused::class.java,
-              ApplicationActuationResumed::class.java,
-              ApplicationActuationPaused::class.java,
-              ResourceValid::class.java,
-              ResourceCreated::class.java
-            )
+      context("resource status") {
```

Comment:
I realized we didn't have tests for turning events or other information from the data store into resource statuses, so I added these.